src/components/chips/js/contactChipsDirective.js

```diff
@@ -63,6 +65,8 @@ var MD_CONTACT_CHIPS_TEMPLATE = '\
               md-items="item in $mdContactChipsCtrl.queryContact($mdContactChipsCtrl.searchText)"\
               md-item-text="$mdContactChipsCtrl.itemName(item)"\
               md-no-cache="true"\
+              md-min-length="$mdContactChipsCtrl.minLength"\
+              md-has-not-found="true"\
```

Comment:
Sorry for the delay; Thomas and I have both been traveling a bit.

Can you describe the necessity of the `md-has-not-found` here since it is automatically added if we see the associated template?